dev/inventory

```diff
@@ -1,13 +1,5 @@
 [is]
-is_1 ansible_host=[ip_address] ansible_user=[ssh_user]
+is_1 ansible_host=172.28.128.5 ansible_user=vagrant
```

Comment:
Shall we change this to place holders?

Reply:
Fixed https://github.com/wso2/ansible-is/pull/45/commits/cf8983700c30d2f4374691203bd4f7d737dd1b11